docs/user/whats-new.asciidoc

```diff
@@ -74,8 +74,11 @@ save a dashboard without ending your editing session. To rename the dashboard or
 [[runtime-fields-7-12]]
 === Runtime fields in Discover
```

Comment:
Maybe this summary should be a little more detailed, I gave a one-sentence example above. The thing that we're getting questions about is "why did the format of Discover change", and "why am I seeing duplicate information", and I think we can simplify the explanation.